doc/rst/source/makecpt.rst

```diff
@@ -313,6 +319,15 @@ we always get a color regardless of the *z* value, try
 
     gmt makecpt -Cjet -T0/500 -Ww > wrapped.cpt
 
+To build a categorical table with 3 categories and add specific category
+names to them, try::
+
+    gmt makecpt -Ccubhelix -T0/3/1 -F+cClouds,Trees,Water > cat.cpt
+
+To instead add unique category labels A, B, C, ... to a 10-item categorical CPT, try::
+
+    gmt makecpt -Cjet -T0/q0/1 -F+cA
```

Comment:
Typo here? `q0` should be `10`?

Reply:
yes thanks. I updated this directly in master.